docs/_sources/meta.rst.txt

```diff
@@ -143,6 +143,9 @@ The ``noarch: python`` can be used to build pure Python packages:
 * and do not use selectors to ``skip`` building the recipe on a specific platform or
   for a specific version of python (e.g. ``skip: True  # [py2k]``).
 
+* don't have a ``scripts`` dir, all ``noarch`` recipes which have command line
+  tools need to use ``entry_points``
```

Comment:
I don't know that this is true.

Reply:
If it is, can you provide more of an explanation

Reply:
please

Reply:
I think this is a cb2/cb3 issue. https://github.com/conda-forge/conda-forge.github.io/issues/486#issuecomment-345001914

Reply:
I don't know that is enough evidence to make it to the docs.

Reply:
I think there was a better issue, I'm trying to find it.

Reply:
Here was the thing I was looking for https://github.com/conda/conda-build/pull/2672

Reply:
I guess this should be moot now.